Source/SimCacheModule/Source/Module/SimCacheModule.cpp

```diff
@@ -34,15 +35,22 @@ bool SimCacheModule::Initialize()
 		return false;
 	}
 
+	const bool InitializeTrackerViewModelSucceeded = InitializeTrackerViewModel();
+	if ( !InitializeTrackerViewModelSucceeded )
+	{
+		return false;
+	}
+
 	return true;
 }
 
 // -----------------------------------------------------------------------------
 
 void SimCacheModule::Uninitialize()
 {
-	UninitializeSimConnectClient();
+	UninitializeTrackerViewModel();
```

Comment:
I think this is fine, but I'm just wondering if there's a particular reason for the order/sequence here?

Reply:
Uninitializing in the reverse order they were initialized in, so if any of them use another as a dependency the user of the system is destroyed before the system itself